generators/node/basicGenerator.js

```diff
@@ -292,7 +292,7 @@ const calcValue = function(sampleDef, sampleName, freq, data, prevSample) {
     }
     const range = Math.abs(rangeDef.range[0] - rangeDef.range[1]);
     let val = Math.random() * range + rangeDef.range[0] + rangeDef._offset;
-
+    	
```

Comment:
No need to create an additional format 'int'. Simply use format "0."
The readme points to the numeral package in https://github.com/MicroFocus/ColorYourData/tree/master/generators/node#formatting-numerical-values.

See http://numeraljs.com/ for details

Reply:
I used 'int' because, in basicGenerator, there is a format "int" mentioned for generating integers as below.
  if (rangeDef.format) {
      if (rangeDef.format === 'int') {
        val = Math.round(val + 0.5);
      } else {
        val = numeral(val).format(rangeDef.format);
      }

Reply:
Instead of format "int", as you said, the following also generates integers.
"numberOfCritical" : {"range": [0,10],"format": "0"},

Reply:
I've overseen 'int'. I thought you added it.
The format is "0." with a trailing dot. Have you checked numeraljs?
int is fine too.

But I still see changes in basicGenerator.js